ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -3566,4 +3566,52 @@ AHU_US_SVL_MAT2_4:
   - supply_fan_run_command
   - supply_fan_speed_frequency_sensor
   - supply_fan_speed_percentage_command
-  - supply_air_flowrate_sensor
\ No newline at end of file
+  - supply_air_flowrate_sensor
+  
+  
+AHU_UK_LON_X1_1:
```

Comment:
A few notes:
1. Why dont we have any information for the supply/exhaust fans (command/status/speeds), orr any outside air information?
2. Do not use the name of the site in the type name.